### PR TITLE
Fixed description in deploy scripts

### PR DESCRIPTION
**Pull request by susuhahnml**
### Changes in the command line

- **New single output argument `--out`**: This removes flags: `--render` `--tex` `--gif`.
   - `out=facts`: Prints preprocessed facts
   - `out=dot`: Prints string in DOT language
   - `out=render`: Generates images with the rendering method of graphviz
   - `out=gif`: Generates a gif after rendering
   - `out=tex`: Prints a latex file
- **Deterministic graphs**: The generated graphs will now be more stable on the way they are printed from one run to another. Addinitionally the new `--seed` argument allows to set the seed used in clingo and in the engine for rendering.
- **Save oputput**: The facts, dot and tex are now outputed to stdout and can be saved in files using argument `--save`
- **New name format for output**: The new `--name-format` argument sets the format of the name of the oputput file and can include `{graph_name}` and `{model_name}`. This removes the older `--name-prefix` argument.
- **Model selection**: The model selection now starts on index `0`
- **Gifs for multi model**: Now gifs can be generated for multiple models
- **Gifs params**: The argument `--gif-param` is removed and the argument `--fps` was added
- **Sorting gif images**. The images of a gif can now be sorted with the `--sort` argument
- **Visualization encoding**: The visualization encoding can now also be passed as an argument `--viz-encoding` and will be applied to multiple models if a json is piped.

### Changes in the python package

- **Clingraph is now Factbase**: The Clingraph class was replaced by the new class is Factbase with the same functionality to add the facts and strings. This calss doesn't have the `compute_graphs` and `save` methos. This are now part of the gaphviz subpackage.
- **Graphviz subpackage**: The graphviz subpackage ``clingraph.graphviz`` now has the standalone functions `compute_graphs`, `dot` and `render`
- **Graphviz subpackage tex**: The graphviz subpackage ``clingraph.graphviz.tex`` now has the standalone functions `tex` to generate latex code
- **Graphviz subpackage imageio**: The graphviz subpackage ``imageio`` now has the standalone functions `save_gif` to generate gifs
- **MultiModelClingraph removed**: This class was removed and now functions accept a list as input for multi models or a dictionary for single models.

### Bugs

- **Escaped characters**: Now escaped characters are parsed differently. The json integration for escaped characters is pending fixes of clasp json output.

### CODE REVIEW

File: .github/deploy.yml [hunk 1]
```diff
@@ -55,7 +55,8 @@ conda:
         license: MIT
         license_family: MIT
         license_file: LICENSE
-        summary: ORM interface for the Clingo ASP Solver
+        summary: A visualizer for graphs defined as a set of facts with special integration for clingo
```
Comment (susuhahnml) [inline .github/deploy.yml:58]: @rkaminsk Can I change this manually also in `.github/conda/meta.yml` or is that one of the generated files? When are these files generated?

File: .github/trigger.sh [hunk 2]
```diff
@@ -32,7 +32,7 @@ case $1 in
         fi
         wip=false
         branch=$2
-        ;&
+        ;;
```
Comment (rkaminsk) [inline .github/trigger.sh:35]: This breaks the script. You can no longer deploy releases. You are hit by <https://en.wikipedia.org/wiki/GNU_General_Public_License#Legal_barrier_to_app_stores>.  Apple won't update bash to version 4.0. You could use the zsh instead.

File: .github/deploy.yml (hunk 1 above)
Comment (rkaminsk) [reply .github/deploy.yml:58]: No, this script is generated using <https://github.com/potassco/deploy> (a private repository).

**Review by rkaminsk — CHANGES_REQUESTED** [review]
Can you test the script with `/bin/zsh` or `zsh trigger.sh` if Apple uses different paths. I just run the deploy script.

File: .github/deploy.yml (hunk 1 above)
Comment (susuhahnml) [reply .github/deploy.yml:58]: Ok, I see the right file to change this was the `conda/meta` how do I run this private script to update files like this `deploy.yml`?

Comment (rkaminsk) [reply .github/deploy.yml:58]: You just check out the repository. And run `python <path-to-repository>/deploy.py` in clingraph's .github folder. You will rarely have to do this. It's mostly me who updates all the projects if some CI script breaks.

File: .github/trigger.sh (hunk 2 above)
Comment (susuhahnml) [reply .github/trigger.sh]: Done

**Pull request merged**